MinPlatformPkg/Test/Library/TestPointCheckLib/TraditionalMmTestPointCheckLib.c

```diff
@@ -415,17 +378,14 @@ TestPointSmmReadyToBootSmmPageProtectionHandler (
 
 /**
   Dispatch function for a Software SMI handler.
-
   Caution: This function may receive untrusted input.
   Communicate buffer and buffer size are external input, so this function will do basic validation.
-
   @param DispatchHandle  The unique handle assigned to this handler by SmiHandlerRegister().
   @param Context         Points to an optional handler context which was specified when the
                          handler was registered.
   @param CommBuffer      A pointer to a collection of data in memory that will
                          be conveyed from a non-SMM environment into an SMM environment.
   @param CommBufferSize  The size of the CommBuffer.
-
   @retval EFI_SUCCESS Command is handled successfully.
 **/
 EFI_STATUS
```

Comment:
Ditto for this function?

Reply:
I think the code below this line all have the same comment. I guess a general question for them is why they are separate implementations?

Reply:
Consolidated.